src/pages/index.js

```diff
@@ -0,0 +1,327 @@
+  const previewModalImage = previewModal.querySelector('.modal__image');
+  const previewModalCaption = previewModal.querySelector('.modal__caption');
+  const previewCloseButton = previewModal.querySelector('.modal__close-btn_type_preview');
+
+  //Avatar Modal
+  const editAvatarModal = document.getElementById('edit-avatar-modal');
+  const editAvatarCloseButton = document.querySelector('#edit-avatar-close-btn');
+  const editAvatarForm = editAvatarModal.querySelector('form');
+  const editAvatarLinkInput = editAvatarModal.querySelector('#edit-avatar-link-input');
+  const avatarSubmitBtn = editAvatarModal.querySelector('.modal__submit-btn');
+  const profileAvatar = document.getElementById('profile-avatar');
+
+  function closeAvatarModal() {
+    closeModal(editAvatarModal);
+  }
+
+  const openAvatarModal = () => {
+    resetValidation(editAvatarForm, [editAvatarLinkInput]);
+    openModal(editAvatarModal);
+  };
+
+
+  function handleEditAvatarSubmit(evt) {
+    evt.preventDefault();
+
+    if (!editAvatarForm.checkValidity()) {
+      return;
+    }
+
+    const avatarUrl = editAvatarLinkInput.value;
+
+    avatarSubmitBtn.textContent = 'Saving...';
+    avatarSubmitBtn.disabled = true;
+
+    api.editUserAvatar(avatarUrl)
+      .then((userData) => {
+        // Ensure that userData includes the correct avatar URL
+        profileAvatar.src = userData.avatar;
+
+        closeAvatarModal();
+        avatarSubmitBtn.textContent = 'Save';
+        avatarSubmitBtn.disabled = false;
+      })
+      .catch((err) => {
+        console.error('Error updating avatar:', err);
+
+        avatarSubmitBtn.textContent = 'Save';
+        avatarSubmitBtn.disabled = false;
+      });
+  }
+
+
+
+  // Delete Modal
+  const deleteModal = document.querySelector('#delete-card-modal');
+  const deleteModalConfirmButton = deleteModal.querySelector('.modal__delete-btn');
+  const deleteModalCancelButton = deleteModal.querySelector('.modal__cancel-btn');
+  const deleteModalCloseButton = deleteModal.querySelector('.modal__close-btn_type_delete');
+
+  let cardToDelete = null;
+
+  function openDeleteModal(cardElement) {
+    cardToDelete = cardElement;
+    openModal(deleteModal);
+  }
+
+
+  function handleDeleteCard() {
+    if (cardToDelete) {
+      const cardId = cardToDelete.dataset.cardId;
+
+      api.deleteCard(cardId)
+        .then(() => {
+          cardToDelete.remove();
+          closeModal(deleteModal);
+        })
+        .catch(err => {
+          console.error('Error deleting card:', err);
+        });
+    }
+  }
+
+
+  // Cards
+  const cardTemplate = document.querySelector('#card-template').content;
+  const cardsContainer = document.querySelector('.cards__list');
+
+  api.getAppInfo()
+    .then(([userInfo, cards]) => {
+      document.querySelector('.profile__name').textContent = userInfo.name;
+      document.querySelector('.profile__description').textContent = userInfo.about;
```

Comment:
`document.querySelector('.profile__name').`
`document.querySelector('.profile__description')`


You've found these element in lines 27-28